### PR TITLE
Feat/display farmers statistic

### PR DESCRIPTION
**Pull request by abidex4yemi**
# Description

* Display all farmers statistic

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -18,6 +18,6 @@
     REACT_APP_API_URL = "https://t-ndo.herokuapp.com"
 
 # Deploy Preview context: All deploys generated from a pull/merge request
-# will inherit these settings. 
+# will inherit these settings.
 [context.deploy-preview.environment]
-  REACT_APP_API_URL = "https://tndo-temp-staging.herokuapp.com"
\ No newline at end of file
+  REACT_APP_API_URL = "https://tieme-ndo-backend-staging1.herokuapp.com"
```
Comment (AlexEntrepreneur) [inline netlify.toml]: Not sure whether the original URL staging should be changed as this is our main one. @Pav0l ?

File: src/components/pages/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -23,6 +28,29 @@ const Dashboard = ({ farmers, history }) => {
     }
   }, [farmers]);
 
+  React.useEffect(() => {
```
Comment (AlexEntrepreneur) [inline src/components/pages/Dashboard/Dashboard.js]: Could destructure `useEffect` from `React` to keep the codebase consistent. 👍

File: src/components/pages/Dashboard/Dashboard.js [hunk 3]
```diff
@@ -23,6 +28,29 @@ const Dashboard = ({ farmers, history }) => {
...
+    axiosWithHeader()
+      .get(pathObj.getFarmersStatistic)
```
Comment (AlexEntrepreneur) [inline src/components/pages/Dashboard/Dashboard.js]: Here it is better to create a new helper to keep the codebase consistent 👍

File: src/components/partials/FarmersStatistic.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Statistic, Container, Segment } from 'semantic-ui-react';
+
+const FarmersStatistic = props => {
+  return (
+    <Container style={{ marginBottom: '40px' }}>
+      <Segment.Group horizontal>
+        <Segment textAlign="center" style={{ padding: '14px 0 0 0' }}>
```
Comment (AlexEntrepreneur) [inline src/components/partials/FarmersStatistic.js]: Here we could use padding and margin to make the stat always centered: `style={{ padding: '14px 0', margin: 'auto 0' }}`

File: src/components/partials/FarmersStatistic.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+          <Statistic
+            label="Total Farmers"
+            value={`${props.totalNumOfFarmers}`}
+            color="teal"
+            size="tiny"
```
Comment (AlexEntrepreneur) [inline src/components/partials/FarmersStatistic.js]: I think `small` would be a closer size to the design files. 👍

File: netlify.toml (hunk 1 above)
Comment (Pav0l) [reply netlify.toml]: Yeah, this should be changed when merging the PR. This URL was used only for QA on different staging server (the main one was used by other feature team)

**Pull request merged**